src/lib/serviceRoutines/postRegisterContext.cpp

```diff
@@ -163,6 +165,32 @@ std::string postRegisterContext
 )
 {
   RegisterContextResponse  rcr;
+  std::string              answer;
+
+  //
+  // If more than ONE service-path is input, an error is returned as response.
+  // If NO service-path is issued, then the default service-path "/" is used.
+  // After these checks, the service-path is checked to be 'correct'.
+  //
+  if (ciP->servicePathV.size() > 1)
+  {
+    LM_W(("Bad Input (more than one service path for a registration)"));
+    rcr.errorCode.fill(SccBadRequest, "more than one service path for notification");
+    answer = rcr.render(RegisterContext, ciP->outFormat, "");
+    return answer;
+  }
+  else if (ciP->servicePathV.size() == 0)
+  {
+    ciP->servicePathV.push_back("/");
```

Comment:
I have found it, at rest.h:

```
define  DEFAULT_SERVICE_PATH            "/"
```

Not sure if that is the best place or if it is better to move to a different .h ....